src/styles/main.css

```
@@ -1,6 +1,7 @@
 @import 'tailwindcss';
 
 @plugin '@tailwindcss/typography';
+@plugin 'daisyui';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)